docs/Sprints/planning/sprint7.md

```diff
@@ -0,0 +1,72 @@
+# Planejamento Sprint 7
+
+**Período: 11/10/2020 a 17/10/2020**<br>
+**Data da Reunião: 10/10/2020**
+
+## Issues
+
+| Número | [Issue](Modeling/objeto?id=Issue) | Pontuação | Participantes |
+|--------|-----------------------------------|-----------|---------------|
+| [**#129**](https://github.com/UnBArqDsw/2020.1_G12_Stock/issues/129) | Refatoração dos Diagramas com base nos Padrões de Projeto | 8 | Micaella Gouveia e Sofia Patrocínio |
+| [**#145**](https://github.com/UnBArqDsw/2020.1_G12_Stock/issues/145) | Refatorar Diagramas com base no Feedback da professora | 5 | Todos |
+| [**#146**](https://github.com/UnBArqDsw/2020.1_G12_Stock/issues/146) | Refatorar Diagrama Entidade e Relacionamento | 3 | Gabriel Alves e Pedro Igor |
+| [**#147**](https://github.com/UnBArqDsw/2020.1_G12_Stock/issues/147) | Implementar Hierarquia de Usuários com base no Factory Method | 8 | Micaella Gouveia e Sofia Patrocínio |
+| [**148**](https://github.com/UnBArqDsw/2020.1_G12_Stock/issues/148) | Implementar Classificação de Produtos (perecível e não perecível) com base no Factory Method | 8 | Gabriel Alves e Pedro Igor |
+| [**149**](https://github.com/UnBArqDsw/2020.1_G12_Stock/issues/149) | Implementar Classe Base para implementação do Decorator (com métodos cadastrar, listar e remover) | 8 | Gabriel Davi |
+
+
+
+## Pontuação: 40
+## Quadro de Distribuição de Pontos
+
+Este quadro divide a pontuação das [issues](Modeling/objeto?id=Issue) com a quantidade de participantes, para ter a noção da distribuição de pontos por pessoa, evitando a sobrecarga de tarefas para algum membro.
+
+| Participante | Distribuição por Pontuação de [Issue](Modeling/objeto?id=Issue) e Partipantes | Pontuação Média |
+|--------------|-------------------------------------------------------------------------------|-----------------|
+| Gabriel Alves | 1.5 + 4 + 1 | 7 |
+| Gabriel Davi | 8 + 1 | 9 |
+| Micaella Gouveia | 4 + 4 + 1 | 9 |
+| Pedro Igor | 1.5 + 4 + 1 | 7 |
+| Sofia Patrocínio | 4 + 4 + 1 | 9 |
+
+## Data de Entrega para Issues
+
+Nesta Sprint, alguns documentos são pré-requisitos de outros, sendo necessário que eles sejam entregues antes. Estipulamos datas de entrega para esses documentos a fim de que as outras entregas não sejam prejudicadas.
+
+
+| Número | [Issue](Modeling/objeto?id=Issue) | Data de Entrega |
+|--------|-----------------------------------|-----------------|
+| [**#145**](https://github.com/UnBArqDsw/2020.1_G12_Stock/issues/145) | Refatorar Diagramas com base no Feedback da professora | 5 | 12/10/2020 |
+
+
+## Relato
+* Nesta reunião, cada membro do grupo falou um pouco sobre os padrões que projeto que estudou, fizemos uma rotação nos estudos e evoluimos o documento de [Estudo Dirigido](Project/EstudoDirigido.md).
+
+| Padrão de Projeto | Detalhes | Responsável |
+|--------|----------|-------------|
+| GRASP | Criador, Especialista e demais | Micaella Gouveia |
```

Comment:
Aqui tem que rotacionar as pessoas para a próxima semana. Ou se não for dividir os estudos essa semana, não colocar essa tabela, pois parece que vai ser o estudo dessa semana.

Reply:
Essa semana não teremos mais issues de estudo, retirei a tabela